modules/rostests/win32/user32/serverside/serverside.c

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
This entire file can just be added as a normal apitest?
That way it will be ran automatically at each testrun.